### ISSUE TITLE
Some YouTube videos do not have history recorded in Payments

### ISSUE THREAD

**Issue by NejcZdovc** [brave/browser-laptop#12318]
### Description

some YT videos (not all) do not display in ledger table. I'm seeing `Error while getting publisher from media Error: invalid url parameter` when I have ledger_verbose on.

### Steps to Reproduce

  1.
  2.
  3.

**Actual result:**

**Expected result:**

**Reproduces how often:**

### Brave Version

**about:brave info:**

**Reproducible on current live release:**

**Comment by NejcZdovc** [comment]
with the 2.0.2 version of `bat-publisher` we are receiving different errors now.

STR

1) visit https://www.youtube.com/watch?v=RtU_mdL2vBM on #12319 branch
2) you get 
```
Error while getting publisher from media Error: HTTP response 404 for GET //yt3.ggpht.com/s900-nd-c-c0xffffffff-rj-k-no/AK162_6PBbkOM0Ee3CHcxATjuMvIX4vJLxpfPK9VEg
```

**Comment by NumDeP** [comment]
Hi @NejcZdovc I noticed you're currently implementing a fix but I was wondering if you can apprehend this comment and either merge/amend or create a new issue for it for the current release or the next.

I happened to notice this issue yesterday and realized that after a period of time in a particular website or channel, the Ledger part of the system/tech didn't pick it up.
I was practically on youtube.com/user/chomskysphilosophy for over half-an-hour and re-opened a video in another tab and it still didn't record it.

One other thing 'Time Spent' column shrunk, maybe because I've got a smaller screen desktop but this didn't occur before the last update.

If you'll allow me to also squeeze in a question and a couple of changes, are there plans to reorganize the menu within brave?
I mention this because there are several aspects such as getting to about:preferences#payments efficiently which would be good and simplified not to mention internal changes within the Brave Payments such as:

- a shoot-up button on the bottom-right

- an undo & restore notification if a publisher is accidentally deleted (not that it has happened to me yet)

- After pinging (if that makes sense) a publisher, it should follow it upwards with % on for entering

Thanks @NejcZdovc

**Comment by NejcZdovc** [comment]
>One other thing 'Time Spent' column shrunk, maybe because I've got a smaller screen desktop but this didn't occur before the last update.
We completely changed code in 0.20 which should fix all this CSS problems

>a shoot-up button on the bottom-right
Can you please explain a little bit more what you mean by this

> an undo & restore notification if a publisher is accidentally deleted (not that it has happened to me yet)
We already have undo functionality. You can delete one publisher and then undo it in preferences->shields->saved site exceptions->Brave payments

>After pinging (if that makes sense) a publisher, it should follow it upwards with % on for entering
so after you pin a publisher you would like to focus percentage input for that publisher you just pinned

**Comment by NumDeP** [comment]
Thanks for the reply @NejcZdovc 

That shoot-up button which I might have confused you with is something many website with quite a lot of content have embedded on the bottom-right hand side of their websites which basically automatically speeds to the top of the page once clicked so users don't have to manually scroll or use the scroll bar to push upwards. I've realized that this could be useful considering the amount of publishers that are automatically stored and after pinging selected publishers to divest a better percentage to selected publishers you might have noticed you continuously have to keep scrolling up and down.

I only stipulated the reason for a temporary undo notification to appear if an accidental change like I mentioned occurred in case a user whilst pinning or entering a percentage in Action deletes a publisher, it doesn't seem fitting for Brave to store deleted publishers but maybe I'm wrong and is useful to more users than I realize. 

Your last question is also related to the shoot-up button feature. So having many publishers and channels in about:preferences#payments available some publishers/channels because a percentage hasn't been set can appear right at the bottom of the payment page, after finding the particular publisher/channel, pinning it makes that particular publisher naturally go to the top but users have to manually scroll to the top every time, so either the shoot-up button feature can be useful here or after pinning the publisher it should allow the % box to be open as well as going to the top of the page where that publisher is.
